packages/modal/src/Modal.js

```diff
@@ -43,7 +44,12 @@ const Overlay = styled(DialogOverlay)`
   ${props => `
     z-index: ${props.zindex || 100};
     font-family: ${props.theme.font};
+    line-height: ${props => props.theme.lineHeights.standard};
```

Comment:
Should we be concerned about keeping this in sync with <Base /> from ThemeProvider?

https://github.com/pricelinelabs/design-system/blob/c2ad5c588944d5713bb6c61f32ce20a81368d31b/packages/core/src/ThemeProvider.js#L6-L14

Reply:
personally, I feel it's fine since it's grabbing things from the theme,

Reply:
Ok, that sounds fair to me